apps/src/aichat/views/modelCustomization/SetupCustomization.tsx

```diff
@@ -127,6 +128,34 @@ const SetupCustomization: React.FunctionComponent = () => {
     );
   };
 
+  const sliderProps = {
+    name: 'temperature-slider',
+    value: aiCustomizations.temperature,
+    minValue: MIN_TEMPERATURE,
+    maxValue: MAX_TEMPERATURE,
+    step: SET_TEMPERATURE_STEP,
+    hideValue: true,
+    disabled: isDisabled(temperature) || readOnlyWorkspace,
+    onChange: event => {
+      const value = parseFloat(event.target.value).toFixed(1); // Round to nearest tenth.
+      dispatch(
+        setAiCustomizationProperty({
+          property: 'temperature',
+          value: value,
+        })
+      );
+    },
+    className: styles.temperatureSlider,
+    leftButtonProps: {
+      icon: {iconName: 'minus', title: 'Decrease'},
+      ['aria-label']: 'Decrease',
+    },
+    rightButtonProps: {
+      icon: {iconName: 'plus', title: 'Increase'},
+      ['aria-label']: 'Increase',
+    },
+  } as SliderProps;
```

Comment:
Why is the cast needed?

Reply:
Oh yeah, I meant to type, not typecast - updated!